bundles/framework/layerlist/view/LayerViewTabs/SelectedLayers/SelectedTab.jsx

```diff
@@ -21,11 +21,10 @@ const StyledBadge = styled.div`
     background: #ffd400;
     border-radius: 4px;
     text-align: center;
-    padding: 2px 10px;
```

Comment:
Is there a reason padding was removed? Now it looks like 
![image](https://user-images.githubusercontent.com/2210335/114938475-d21e4d80-9e47-11eb-8bee-4304bf2d7120.png)

While with the padding it looks like 
![image](https://user-images.githubusercontent.com/2210335/114938513-dc404c00-9e47-11eb-9b71-0a55b0db2b42.png)

Reply:
Oh no there is no reason. This was my mistake. I will correct it. Thanks!